templates/cluster/gcp-standalone-cp/templates/machinedeployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: MachineDeployment
+metadata:
+  name: {{ include "machinedeployment.name" . }}
+  annotations:
+    # Temporary fix to address https://github.com/k0sproject/k0smotron/issues/911
+    machineset.cluster.x-k8s.io/skip-preflight-checks: "ControlPlaneIsStable"
+spec:
+  clusterName: {{ include "cluster.name" . }}
+  replicas: {{ .Values.workersNumber }}
+  selector:
+    matchLabels:
+      cluster.x-k8s.io/cluster-name: {{ include "cluster.name" . }}
+  template:
+    metadata:
+      labels:
+        cluster.x-k8s.io/cluster-name: {{ include "cluster.name" . }}
+    spec:
+      version: {{ regexReplaceAll "\\+k0s.+$" .Values.k0s.version "" }}
```

Comment:
nit: i think we could simplify it with the `{{ (split "+" .Values.k0s.version)._0 }}` (reference the [templates/cluster/docker-hosted-cp/templates/machinedeployment.yaml:22](https://github.com/k0rdent/kcm/blob/bca47271c04adf6644920c49ffcf8fd590aace60/templates/cluster/docker-hosted-cp/templates/machinedeployment.yaml#L22))